curvefs/src/client/fuse_client.cpp

```diff
@@ -1149,8 +1149,38 @@ CURVEFS_ERROR FuseClient::FuseOpSetXattr(fuse_req_t req, fuse_ino_t ino,
     return CURVEFS_ERROR::OK;
 }
 
+CURVEFS_ERROR FuseClient::FuseOpRemoveXattr(fuse_req_t req, fuse_ino_t ino,
+    const char *name) {
+    std::string strname(name);
+    VLOG(1) << "FuseOpRemoveXattr ino: " << ino << ", name: " << name;
+
+    if (option_.fileSystemOption.disableXAttr || IsSpecialXAttr(name)) {
+        return CURVEFS_ERROR::NODATA;
```

Comment:
is CURVEFS_ERROR::NODATA appropriate?

normally, user can only add/remove/modify "user.\*" xattr in my mind. in curvefs, maybe some "curve.*" in addtion. 

Some error code should be permission denied?

Reply:
fixed